examples/benchmark/ocis-ldap/ldif/create-ldif.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+# USAGE: ./create-ldif.sh 100 > users.ldif
+
+NUM=1
+
+echo "# This LDIF file contains examples of groups and users that could be created
+# in LDAP when testing. It is not used by the automated tests. The automated
+# tests create LDAP groups and users on-the-fly.
+#
+# Examples here might be useful if you are manually setting up some LDAP entries
+# for local development and testing.
+dn: ou=TestGroups,dc=owncloud,dc=com
+objectclass: top
+objectclass: organizationalUnit
+ou: TestGroups
+
+
+dn: ou=TestUsers,dc=owncloud,dc=com
+objectclass: top
+objectclass: organizationalUnit
+ou: TestUsers
+
+"
+
+while [ $NUM -lt $1 ]
```

Comment:
```suggestion
while [ $NUM -le $1 ]
```

we only got 299 users, and we want 300